v2/api-validator/tests/server-tests/bad-request-body.test.ts

```diff
@@ -41,6 +51,11 @@ describe('Test request bodies missing one required property', () => {
           beforeAll(async () => {
             const badBody = _.cloneDeep(goodBody);
             deleteDeepProperty(badBody, propertyPath);
+
+            if (Object.prototype.hasOwnProperty.call(badBody, 'idempotencyKey')) {
+              badBody['idempotencyKey'] = randomUUID();
+            }
```

Comment:
I'd expect it to be something like this:
```javascript
if (badBody.hasOwnProperty('idempotencyKey')) { ... }
```
Is there something that prevents using this simpler form? Looks like `badBody` is a proper object.

Reply:
See reply on other MR